ui/admin/app/styles/app.scss

```diff
@@ -452,3 +457,100 @@
     }
   }
 }
+
+// secret editor
```

Comment:
Commenting with no context:
In [this file](https://github.com/hashicorp/boundary-ui/pull/1499/files#diff-3b83e0b7d2d1b6c3989f81f6450624eb7a3b70a7cf5c65d886a0449a266b5816R1) I understand `<div class='secret-editor'>` as the root markup of our editor. Is that right?

If is not right, ignore the rest of the comment.

If so, for dev experience and facilitate understanding of the code, what do you think of wrapping all the css regarding the editor within `.secret-editor` ?

Something like:
```css
.secret-editor {
  ...css code

  .json-secret-editor { ... }

and so on
}
```

This way helps to understand all the code within `.json-secret-editor` pertains to the editor and helps isolating that css to avoid override collisions.

What do you think?

Reply:
That is actually how I originally had it before I ever opened the PR but I wasn't sure if that was desired or not. I think we could discuss that during tech time to get the whole team's input.

Reply:
Good point! this is not like a code rule, more developer experience so good call to discuss it on FE tech time 😉